boards/nxp/imx93_evk/board.yml

```diff
@@ -1,6 +1,6 @@
 board:
   name: imx93_evk
-  full_name: i.MX93 EVK
+  full_name: i.MX93-EVK
```

Comment:
```suggestion
  full_name: i.MX93EVK
```
see https://www.nxp.com/design/design-center/development-boards-and-designs/i.MX93EVK

Reply:
Or should it be "MCIMX93-EVK" ;) nxp.com:
![image](https://github.com/user-attachments/assets/4eee8d6d-a5fb-4af6-8ef8-ab9876d926bd)